noxfile.py

```diff
@@ -63,31 +66,6 @@ def install_torch_trt(session):
         session.run("python", "setup.py", "develop")
 
 
-def download_datasets(session):
```

Comment:
@narendasan @peri044 : Why are we dropping download_datasets in nox?

Reply:
This was changed as part of @peri044's cos similarity overhaul